<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>653</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>653</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/systray-transparent-32x32.png</normaloff>:/images/systray-transparent-32x32.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="confList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>76</width>
          <height>79</height>
         </size>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="spacing">
         <number>17</number>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <item>
         <property name="text">
          <string>General</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/settings/parameter.png</normaloff>:/images/settings/parameter.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Color</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/settings/color.png</normaloff>:/images/settings/color.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Server</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/settings/network.png</normaloff>:/images/settings/network.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="confStack">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="General">
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>511</width>
            <height>451</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="generalLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>General</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_5">
             <property name="labelAlignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="formAlignment">
              <set>Qt::AlignCenter</set>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Language:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="languageComboBox">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>English</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Fran√ßais</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="defaultGroupBox">
             <property name="title">
              <string>Default Boxes</string>
             </property>
             <widget class="QWidget" name="formLayoutWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>491</width>
                <height>101</height>
               </rect>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <property name="labelAlignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="formAlignment">
                <set>Qt::AlignCenter</set>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Number of squares:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="squareNumberBox">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>3</number>
                 </property>
                 <property name="maximum">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="customCheck">
             <property name="text">
              <string>Disable Custom Boxes Mode</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="customGroupBox">
             <property name="title">
              <string>Custom Boxes</string>
             </property>
             <widget class="QWidget" name="formLayoutWidget_2">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>491</width>
                <height>101</height>
               </rect>
              </property>
              <layout class="QFormLayout" name="formLayout_2">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <property name="formAlignment">
                <set>Qt::AlignCenter</set>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="customXMLPathLabel">
                 <property name="text">
                  <string>Define the XML Path:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLineEdit" name="customXMLPathLine">
                   <property name="echoMode">
                    <enum>QLineEdit::Normal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="customXMLPathButton">
                   <property name="text">
                    <string>Browse...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="Color">
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>511</width>
            <height>451</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="colorLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Color</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <property name="formAlignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="verticalSpacing">
              <number>40</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>On focus:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>On blur:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_6">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>Opacity (min to max):</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>50</number>
               </property>
               <item>
                <widget class="QSlider" name="colorOpacitySlider">
                 <property name="maximumSize">
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="value">
                  <number>75</number>
                 </property>
                 <property name="sliderPosition">
                  <number>75</number>
                 </property>
                 <property name="tracking">
                  <bool>true</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                 <property name="invertedControls">
                  <bool>false</bool>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBelow</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="colorOpacityLabel">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>50</number>
               </property>
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>20</number>
               </property>
               <item>
                <widget class="QPushButton" name="colorFocusButton">
                 <property name="text">
                  <string>Choose a color</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="colorFocusLabel">
                 <property name="frameShape">
                  <enum>QFrame::WinPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Sunken</enum>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>50</number>
               </property>
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>20</number>
               </property>
               <item>
                <widget class="QPushButton" name="colorBlurButton">
                 <property name="text">
                  <string>Choose a color</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="colorBlurLabel">
                 <property name="frameShape">
                  <enum>QFrame::WinPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Sunken</enum>
                 </property>
                 <property name="lineWidth">
                  <number>1</number>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="Network">
         <widget class="QWidget" name="verticalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>511</width>
            <height>451</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="networkLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Server</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QGroupBox" name="proxyGroupBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="title">
                <string>Server setting</string>
               </property>
               <widget class="QWidget" name="formLayoutWidget_4">
                <property name="geometry">
                 <rect>
                  <x>9</x>
                  <y>19</y>
                  <width>491</width>
                  <height>371</height>
                 </rect>
                </property>
                <layout class="QFormLayout" name="formLayout_4">
                 <property name="fieldGrowthPolicy">
                  <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                 </property>
                 <property name="labelAlignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                 <property name="formAlignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="verticalSpacing">
                  <number>40</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Port:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QSpinBox" name="serverPort">
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="minimum">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <number>65535</number>
                   </property>
                   <property name="value">
                    <number>5900</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="serverPassword">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QCheckBox" name="serverPasswordCheckBox">
                   <property name="text">
                    <string>Require the user to enter this password:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>649</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>customCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>customGroupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serverPasswordCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>serverPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
